deploy/rbd/helm/Chart.yaml

```diff
@@ -4,7 +4,7 @@ appVersion: "1.0.0"
 description: "Container Storage Interface (CSI) driver,
 provisioner, and attacher for Ceph RBD"
 name: ceph-csi-rbd
-version: 0.3.0
+version: 0.4.0
```

Comment:
github doesn't allow me to comment below, but can you also update the branch name